src/main/java/ee/aktors/misp2/configuration/DigiDoc4jConfiguration.java

```diff
@@ -34,8 +35,7 @@
 
 import ee.aktors.misp2.ExternallyConfigured;
 
-import java.io.IOException;
-import java.io.InputStream;
+import java.io.*;
```

Comment:
In the core import statements shouldn't use the `*` notation. Currently, MISP2 doesn't have the same check style rules, but it would be good to follow the same practice despite that.